lib/recording/RecordingManager.js

```diff
@@ -217,6 +218,18 @@ class RecordingManager {
                 logger.warn("[%s-%s] recording  has passed num of clips - stop recording", recordingSession.entryId, recordingSession.recordedEntryId);
                 return true
             }
+
+            if (recordingSession.entryObject.recordingStatus == kalturaTypes.KalturaRecordingStatus.PAUSED)
```

Comment:
it is not really needed to check PAUSED status

Reply:
yes, it is